types/node/crypto.webcrypto.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module "crypto" {
+    // TODO: Define `Crypto`
+    type Crypto = any;
+
+    const webCrypto: Crypto;
+}
```

Comment:
```suggestion
interface NodeCrypto extends Crypto {
  readonly CryptoKey: typeof CryptoKey
}

declare module 'crypto' {
  const webcrypto: NodeCrypto
}
```

Crypto is already defined in TS. This just adds the "webcrypto" (you mistyped the property name, capitalizing the C) property as a crypto export and also adds the CryptoKey class instance.

Note: i'm not taking the [node.js specific extensions](https://nodejs.org/api/webcrypto.html#webcrypto_node_js_specific_extensions) into account here.